Makefile

```diff
@@ -349,15 +349,16 @@ static-web-upload:
 	&& ${MAKEFILE_DIR}/templates/core/terraform/scripts/upload_static_web.sh
 
 build-and-deploy-ui:
-	$(call target_title, "Use Static Web App CLI to deploy UI") \
+	$(call target_title, "Build and deploy UI") \
```

Comment:
does the static-web-upload task need to be conditional on `DEPLOY_UI` ? Need to be able to opt out of it

Reply:
Good point, as the first script will exit? But the following make command will still run?

Reply:
Called the second from the first.